cmake/FindSciPy.cmake

```diff
@@ -4,7 +4,7 @@
 find_package(PythonInterp REQUIRED)
 
 execute_process(COMMAND "${PYTHON_EXECUTABLE}" "-c"
-  "import scipy as s; print(s.__version__); print(s.get_include());"
+  "import scipy as s; print(s.__version__); import numpy as n; print(n.get_include());"
```

Comment:
Thanks for fixing this 👍

Reply:
when is this guaranteed to be the correct behavior?  (i.e. a numpy or scipy version)

Reply:
I'm not sure what you're trying to know. This was the deprecation warning from Python: "<string>:1: DeprecationWarning: scipy.get_include is deprecated and will be removed in SciPy 2.0.0, use numpy.get_include instead"

Reply:
How old of a numpy and/or scipy version would need to be used for the new method to give incorrect answers? Does that help clarify the question?

Reply:
I didn't easily find documentation of this. However, looking through the SciPy source code (in particular the commits to `scipy/__init__.py`, I see that SciPy has been importing NumPy symbols into its namespace since at least 2007. The recent change is to deprecate (and eventually drop) this behavior.

So, I think the new method will always give the right answer.